5.sc_Morphology_Logistic_Regression/0.split_data/scripts/0.split_data_binary_classification.py

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import ast
+import itertools
+import pathlib
+
+import numpy as np
+import pandas as pd
+import pyarrow.parquet as pq
+import toml
+from joblib import dump
+from sklearn.exceptions import ConvergenceWarning
+from sklearn.linear_model import LogisticRegression
+from sklearn.model_selection import GridSearchCV, StratifiedKFold, train_test_split
+from sklearn.utils import parallel_backend, shuffle
+
+# In[2]:
+
+
+# Parameters
+cell_type = "SHSY5Y"
+aggregation = True
+nomic = True
+flag = True
+control = "DMSO_0.100_%_DMSO_0.025_%"
+treatment = "LPS_100.000_ug_per_ml_DMSO_0.025_%"
+
+
+# In[3]:
+
+
+MODEL_TYPE = "binary_classification"
+if flag == False:
+    # read in toml file and get parameters
+    toml_path = pathlib.Path("../1.train_models/single_class_config.toml")
+    with open(toml_path, "r") as f:
+        config = toml.load(f)
+    f.close()
+    control = config["logistic_regression_params"]["control"]
+    treatment = config["logistic_regression_params"]["treatments"]
+    aggregation = ast.literal_eval(config["logistic_regression_params"]["aggregation"])
+    nomic = ast.literal_eval(config["logistic_regression_params"]["nomic"])
+    cell_type = config["logistic_regression_params"]["cell_type"]
+    print(aggregation, nomic, cell_type)
+
+
+# In[4]:
+
+
+path = pathlib.Path(f"../../data/{cell_type}_preprocessed_sc_norm.parquet")
```

Comment:
would recommend `pathlib.path.resolve(strict=True)` if this is an expected input file